test/parallel/test-node-run.js

```diff
@@ -53,6 +57,101 @@ describe('node --run [command]', () => {
     assert.strictEqual(child.code, 0);
   });
 
+  it('chdirs into package directory', async () => {
+    const child = await common.spawnPromisified(
+      process.execPath,
+      [ '--no-warnings', '--run', common.isWindows ? 'cd' : 'pwd'],
+      { cwd: fixtures.path('run-script/sub-directory') },
+    );
+    assert.strictEqual(child.stdout.trim(), fixtures.path('run-script'));
+    assert.strictEqual(child.stderr, '');
+    assert.strictEqual(child.code, 0);
+  });
+
+  it('includes actionable info when possible', async () => {
+    {
+      const child = await common.spawnPromisified(
+        process.execPath,
+        [ '--no-warnings', '--run', 'missing'],
```

Comment:
On line 22, we already check for non-existent scripts. Is this the same?

Reply:
kind of but not really; removing the `it('should emit experimental warning'` would be done when it isn't experimental; i added the check on line 22 since it was checking stderr already but think that test could remove the non-warning checks anyway; i'd be happy to remove the stderr checks for message from that test

Reply:
Sorry it should be line 24 `returns error on non-existent file`

Reply:
I gave the wrong line as a reference

Reply:
no; that line is the cause of needing `common.requireNoPackageJSONAbove();` and tests what happens when it cannot find a `package.json` for a given directory to perform file read -> a JSON parse etc.